### DISCUSSION TITLE
How to disable selecting the first completion in case of ambiguity?

### DISCUSSION THREAD

**Discussion by ittayd** [marlonrichert/zsh-autocomplete#664] [Q&A]
Whether I'm completing file names or arguments (or something else), if I hit `<tab>`, the first choice is completed. I want only the unabmigous prefix to be completed.
I have 

```
zstyle ':autocomplete:*complete*:*' insert-unambiguous yes # tab widgets
zstyle ':autocomplete:*history*:*' insert-unambiguous yes # history search
zstyle ':autocomplete:menu-search:*' insert-unambiguous yes #^s
zstyle ':completion:*' insert-unambiguous true
```

And I also tried
```
unsetopt menu_complete
```

**Comment by marlonrichert** [comment]
It already works like that, but you have a typo in your first block of code. Replace `completion` with `autocomplete` in the last line:
```sh
zstyle ':autocomplete:*' insert-unambiguous true
```

**Reply by alexwforsythe** [reply]
This line is missing from the README (it only has the first 3). It started working once I added this line
